From: Frank Rowand <frank.rowand@am.sony.com>
Subject: [PATCH 2/4] RT: remove conflicting rtc_lock declaration
To: linux-kernel@vger.kernel.org
Cc: mingo@redhat.com, tglx@linutronix.de
Date: Tue, 15 Jan 2008 14:19:55 -0800

From: Frank Rowand <frank.rowand@am.sony.com>

Declaration of rtc_lock in arch/mips/kernel/time.c conflicts with time.h,
remove from include/asm-mips/time.h.

Signed-off-by: Frank Rowand <frank.rowand@am.sony.com>
---
 include/asm-mips/time.h |    2 --
 1 file changed, 2 deletions(-)

Index: linux-2.6.26.3-rt6/include/asm-mips/time.h
===================================================================
--- linux-2.6.26.3-rt6.orig/include/asm-mips/time.h	2008-09-05 00:23:00.000000000 -0400
+++ linux-2.6.26.3-rt6/include/asm-mips/time.h	2008-09-05 00:23:43.000000000 -0400
@@ -19,8 +19,6 @@
 #include <linux/clockchips.h>
 #include <linux/clocksource.h>
 
-extern raw_spinlock_t rtc_lock;
-
 /*
  * RTC ops.  By default, they point to weak no-op RTC functions.
  *	rtc_mips_set_time - reverse the above translation and set time to RTC.
